### DISCUSSION TITLE
How to tell what version of packages Pluto has installed?

### DISCUSSION THREAD

**Discussion by samuela** [fonsp/Pluto.jl#1382] [Questions about using Pluto] [answered]
The new Pluto per-notebook package management is neat, but how do I check what versions of packages Pluto has installed? I tried doing so via `Pkg` but that gives me an error: `UndefVarError: Pkg not defined`.

**Comment by sp94** [comment, marked as answer]
Do you need to run `import Pkg` first? This works for me on Pluto v0.15
```julia
begin
	using PyPlot
	import Pkg; Pkg.status()
end
```
returns
```
      From worker 3:          Status `/tmp/jl_DTpg1D/Project.toml`
      From worker 3:	  [d330b81b] PyPlot v2.9.0
      From worker 3:	  [44cfe95a] Pkg
```

**Reply by samuela** [reply]
Ack, yes... I'm a ding dong!
